dali/base/dadfs.cpp

```diff
@@ -4533,6 +4533,15 @@ protected: friend class CDistributedFilePart;
         dt.setNow();
         setAccessedTime(dt);
     }
+
+    virtual void validate()
+    {
+        if (!existsPhysicalPartFiles(0))
+        {
+            throw MakeStringException(-1, "Physical Parts doesn't exists for logical file !");
+        }
+
```

Comment:
trivial: unnecessary extra newline here and a block with 1 line doesn't normally have open/close braces.
Also, exception text would be better as "Some physical parts do not exists, for logical file : %s", where logicalName was given if set and "[unattached]" was given if logicalName was not yet set.